src/endpoint/s3/ops/s3_put_bucket_lifecycle.js

```diff
@@ -88,7 +88,12 @@ async function put_bucket_lifecycle(req) {
         };
 
         if (rule.ID?.length === 1) {
-            current_rule.id = rule.ID[0];
+            if (rule.ID[0].length > 255) {
```

Comment:
I'm suggesting to move the 255 to a const variable for easier reading of the code and maintenance - in case in the future it will be changed we need to change it in one place only, for example:
`const MAX_LENGTH_RULE_ID = 255` and reuse it in the error message.

Reply:
Done, Thanks